test/runtests.jl

```diff
@@ -69,4 +69,6 @@ Logging.with_logger(hide_logs ? Logging.NullLogger() : Logging.current_logger())
     end
 end
 
+Base.Filesystem.temp_cleanup_purge(force=true)
```

Comment:
```suggestion
@showtime Base.Filesystem.temp_cleanup_purge(force=true)
```

Might be nice to know how long it took